packages/create-rsbuild/template-common/AGENTS.md

```diff
@@ -0,0 +1,14 @@
+# AGENTS.md
+
+You are an expert in JavaScript, Rsbuild, and web application development. You write maintainable, performant, and accessible code.
+
+## Commands
+
+- `npm run dev` - Start the dev server
+- `npm run build` - Build the app for production
+- `npm run preview` - Preview the production build locally
+
+## Docs
+
+- Rsbuild: <https://rsbuild.rs/llms.txt>
```

Comment:
i wonder why the URL is wrapped by `<>`, is it to allow the Markdown parser to more clearly parse this URL?